src/app/sandbox/deploy/LeftSection.tsx

```diff
@@ -96,7 +97,11 @@ export function LeftSection() {
         >
           {isConnected ? 'Deploy' : 'Connect Stacks Wallet'}
         </Button>
-      </HStack>
+        <Text fontSize="xs">
+          *The sandbox doesn't support Clarity 3 deployments. To deploy a Clarity 3 contract, please
+          use Clarinet or the Hiro platform instead.
+        </Text>
+      </Stack>
       <Stack mt={'10px'}>
```

Comment:
Why having flex container with a margin top when it can all be within a flex container with a gap?